watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -1308,6 +1391,12 @@ def calculate_scaling_factors(self):
                 )
                 iscale.set_scaling_factor(self.flow_mol_phase_comp["Liq", j], sf)
 
+            if iscale.get_scaling_factor(self.flow_equiv_phase_comp["Liq", j]) is None:
+                sf = iscale.get_scaling_factor(
+                    self.flow_mol_phase_comp["Liq", j], default=1, warning=True
+                )
+                iscale.set_scaling_factor(self.flow_equiv_phase_comp["Liq", j], sf)
+
```

Comment:
This (and others like this) may be the culprit to the issue of the bad scaling in other tests of models. You are not checking to see if this property is constructed here, so as soon as you touch this variable, the properties get constructed (even for the models that don't use them).

Reply:
@kurbansitterley @aladshaw3 I just hopped on the branch and made a couple fixes to the ion prop model changes. I don't know it'll fix all failed tests, but it should repair at least a couple.

Reply:
Looks like that fixed most of the issues. There are now only 3 failing tests, one of which probably just needs to have the assert changed due to the new set of constraints. The other 2 are bad scaling vars that show up in the `Electrodialysis_0D` and `GAC` models.